esphome/components/waveshare_epaper/waveshare_epaper.cpp

```diff
@@ -250,7 +251,7 @@ void WaveshareEPaperTypeA::dump_config() {
       ESP_LOGCONFIG(TAG, "  Model: 2.9inV2");
       break;
   }
-  ESP_LOGCONFIG(TAG, "  Full Update Every: %u", this->full_update_every_);
+  ESP_LOGCONFIG(TAG, "  Full Update Every: %" PRIu32, this->full_update_every_);
```

Comment:
Extra `%` symbol left here?

```suggestion
  ESP_LOGCONFIG(TAG, "  Full Update Every: " PRIu32, this->full_update_every_);
```